tests/test_git_repository.py

```diff
@@ -497,6 +497,30 @@ def test_get_commits_last_modified_lines_hyper_blame_ignore_hash(depot_tools, tm
         'B.java']
 
 
+@pytest.fixture
+def tmpcwd(tmpdir):
+    cwd = os.getcwd()
+    os.chdir(tmpdir)
```

Comment:
I think you should cast tmpdir to `str` so python 3.4 and 3.5 won't complain. Thanks!

Reply:
Oh, I didn't notice Travis was failing :P

Reply:
Fixed now. Hopefully Travis will pass.